modules/fts/pages/fts-searching-full-text-indexes-aliases.adoc

```diff
@@ -0,0 +1,30 @@
+[#searching-full-text-indexes-aliases]
+== Searching Fulltext Indexes/Aliases
+[abstract]
+Full Text indexes, which are available under the Search tab of the Couchbase Web Console, are another form of the index provided by the Search service.
```

Comment:
Same wording mistakes as mentioned earlier.

Reply:
Not clear. which wording mistakes?

Reply:
Please scroll up charu, 
"are another form of the index provided by the Search service" => this needs to be reworded?
Search service's indexes are accessible mainly over here in UI?

"are another form of the index provided by the Search service" => to me it looks like you have some disconnect in the offerings of search service.. let me know if u want to sync on this..